### PR TITLE
Add details about Softwarepraktikum 2021

### PR DESCRIPTION
**Pull request by jaimergp**
This adds:
* [x] Blog post
* [x] OpenCADD project page that was missing (!)
* [X] PDF of last year for archival
* [x] Slides for this year (WIP)

### CODE REVIEW

File: content/blog/structural-alignment-2021.md
Comment (AndreaVolkamer) [inline content/blog/structural-alignment-2021.md:14]: Todo add link once finished

File: content/education/cadd-seminar.md
Comment (AndreaVolkamer) [inline content/education/cadd-seminar.md:20]:
```suggestion
- Software and project management (60102613): <a href="https://www.mi.fu-berlin.de/bioinf/stud/bachelor/abv/projektmanagement/SWP_projekte/Structural_Alignment.html" target="_blank" class="external">Structural Alignment with OpenCADD WS20/21</a> (continued from WS19/20). [More details about OpenCADD](/projects/opencadd/).
```

File: content/blog/structural-alignment-2021.md
Comment (AndreaVolkamer) [reply content/blog/structural-alignment-2021.md:14]: pdf from 2020 was added to static/files, but not the updated one?

**Review by AndreaVolkamer — APPROVED** [review]
@jaimergp thanks, well done! Small comments added (note the pdf that is currently added to static/files is the one from 2020), can be merged after inclusion without further approval.

File: content/blog/structural-alignment-2021.md
Comment (jaimergp) [reply content/blog/structural-alignment-2021.md:14]: Yes, will upload once we are happy with it!

**Pull request merged**
